tests/bluetooth/ctrl_llcp/src/main.c

```diff
@@ -0,0 +1,301 @@
+/*
+ * Copyright (c) 2020 Demant
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/types.h>
+#include <ztest.h>
+
+/* Kconfig Cheats */
+#define CONFIG_BT_LOG_LEVEL 1
+#define CONFIG_BT_CTLR_COMPANY_ID 0x1234
+#define CONFIG_BT_CTLR_SUBVERSION_NUMBER 0x5678
+
+#define ULL_LLCP_UNITTEST
+
+/* Implementation Under Test Begin */
+#include "ll_sw/ull_llcp.c"
+/* Implementation Under Test End */
+
+struct ull_tx_q tx_q;
+struct ull_cp_conn conn;
+
+u8_t buffer_mem_tx[TX_CTRL_BUF_SIZE * 1];
+u8_t buffer_mem_ctx[TX_CTRL_BUF_SIZE * 1];
+struct mem_pool mem_tx = { .pool = buffer_mem_tx };
+struct mem_pool mem_ctx = { .pool = buffer_mem_ctx };
+const size_t sz_mem_tx = TX_CTRL_BUF_SIZE;
+const u32_t ncount_mem_tx = 1;
+const size_t sz_mem_ctx = PROC_CTX_BUF_SIZE;
+const u32_t ncount_mem_ctx = 1;
+
+void test_init(void)
+{
+	ull_tx_q_init(&tx_q);
+
+	ull_cp_conn_init(&conn);
+	zassert_equal(conn.local.state, LR_STATE_DISCONNECT, NULL);
+
+	conn.tx_q = &tx_q;
+
+	ull_cp_init();
```

Comment:
Move this before `ull_cp_conn_init` ?

Reply:
Yes, that would probably make more sense.